inc/userinjection.class.php

```diff
@@ -104,7 +104,7 @@ public function getOptions($primary_type = '')
         $options['ignore_fields']  = array_merge($blacklist, $notimportable);
 
        //Add displaytype value
-        $options['displaytype']    = ["dropdown"       => [3, 79, 81, 82],
+        $options['displaytype']    = ["dropdown"       => [3, 77, 79, 81, 82],
```

Comment:
Can you please comment on what each ID stands for?
```suggestion
        $options['displaytype']    = [
            "dropdown"       => [
                3,  // ...
                77, // ...
                79, // ...
                81, // ...
                82, // ...
            ],
```